components/feature/script/src/main/java/mozilla/components/feature/script/SlowLoadingFeature.kt

```diff
@@ -0,0 +1,63 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.feature.script
+
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.cancel
+import kotlinx.coroutines.flow.mapNotNull
+import mozilla.components.browser.state.selector.findTabOrCustomTabOrSelectedTab
+import mozilla.components.browser.state.store.BrowserStore
+import mozilla.components.concept.engine.script.SlowScriptCallback
+import mozilla.components.lib.state.ext.flowScoped
+import mozilla.components.support.base.feature.LifecycleAwareFeature
+import mozilla.components.support.base.log.logger.Logger
+import mozilla.components.support.ktx.kotlinx.coroutines.flow.ifChanged
+
+/**
+ * Feature implementation that provides a notification when the
+ * slowScript callback is called and enables the user to
+ * continue or stop running the script.
+ *
+ * @property sessionId id of the session running the slow script.
+ * @property store reference to the application's [BrowserStore].
```

Comment:
```suggestion
 * @param sessionId id of the session running the slow script.
 * @param store reference to the application's [BrowserStore].
```

Parameters are in constructors/methods, properties are in fields. 🙂